src/scribe_data/extract_transform/English/verbs/format_verbs.py

```diff
@@ -0,0 +1,94 @@
+"""
+Format Verbs
+------------
+
+Formats the verbs queried from Wikidata using query_verbs.sparql.
+"""
+
+# pylint: disable=invalid-name
+
+import collections
+import json
+import os
+import sys
+
+LANGUAGE = "English"
+PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+PATH_TO_SCRIBE_DATA_SRC = f"{PATH_TO_SCRIBE_ORG}Scribe-Data/src"
+sys.path.insert(0, PATH_TO_SCRIBE_DATA_SRC)
+
+from scribe_data.load.update_utils import get_path_from_et_dir
+
+file_path = sys.argv[0]
+
+update_data_in_use = False  # check if update_data.py is being used
+if f"{LANGUAGE}/verbs/" not in file_path:
+    with open("verbs_queried.json", encoding="utf-8") as f:
+        verbs_list = json.load(f)
+else:
+    update_data_in_use = True
+    with open(f"./{LANGUAGE}/verbs/verbs_queried.json", encoding="utf-8") as f:
+        verbs_list = json.load(f)
+
+verbs_formatted = {}
+
+all_conjugations = [
+    "presFPS",
+    "presSPS",
+    "presTPS",
+    "presFPP",
+    "presSPP",
+    "presTPP",
+    "pastFPS",
+    "pastSPS",
+    "pastTPS",
+    "pastFPP",
+    "pastSPP",
+    "pastTPP",
+    "pastPart",
+]
+
+for verb_vals in verbs_list:
+    # if infinitive is available add to formatted verbs, else no entry created
+    if verb_vals["infinitive"] not in verbs_formatted.keys():
+        verbs_formatted[verb_vals["infinitive"]] = {}
+
+        infinitive_key = verb_vals["infinitive"]
+        # presFPS
+        verbs_formatted[infinitive_key]["presFPS"] = verb_vals.get("presFPS", "")
+        verbs_formatted[infinitive_key]["presSPS"] = verb_vals.get("presFPS", "")
+
+        # presTPS
+        verbs_formatted[infinitive_key]["presTPS"] = verb_vals.get("presTPS", "")
+
+        # copying over presFPS to remaining present cases
+        verbs_formatted[infinitive_key]["presFPP"] = verb_vals.get("presFPS", "")
+        verbs_formatted[infinitive_key]["presSPP"] = verb_vals.get("presFPS", "")
+        verbs_formatted[infinitive_key]["presTPP"] = verb_vals.get("presFPS", "")
+
+        # assigning simpPast to all past keys if available
```

Comment:
Minor comment that the comment here and other places should be a full sentence. The syntax that we use in Scribe-iOS actually originates in Python :)